src/plugin.json

```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
   "type": "datasource",
-  "name": "Sumo Logic Metrics",
+  "name": "Sumo Logic",
   "id": "sumologic-metrics-datasource",
   "metrics": true,
```

Comment:
I think we should add 
`"logs": true`
https://grafana.com/docs/grafana/latest/developers/plugins/create-a-grafana-plugin/develop-a-plugin/build-a-logs-data-source-plugin/#step-1-enable-logs-support